src/components/slider/slider.st.css

```diff
@@ -0,0 +1,140 @@
+:import {
+    -sb-from: "../../style/default-theme/variables.st.css";
+    -sb-named: color_Border,
+    color_Slider_Track,
+    color_Slider_Track_Hover,
+    color_Slider_Progress,
+    color_Slider_Progress_Hover,
+    color_Slider_Handle,
+    color_Slider_HandleBorder,
+    color_Slider_Handle_Hover,
+    color_Slider_Handle_Active;
+}
+
+:vars {
+    trackHeight: 4px;
+    handleMargin: -6px;
+    handleWidth: 16px;
+    handleHeight: 16px;
+    handleRadius: 8px;
+    handleBorder: 1px solid value(color_Slider_HandleBorder);
+    handleBorderHover: 1px solid value(color_Slider_Handle_Active);;
+}
+
+.root {
+    -st-states: active, disabled, error;
+    width: 100%;
+    position: relative;
+    display: inline-block;
+}
+
+.slider {
+    width: 100%;
+    padding: 15px 0;
+    cursor: pointer;
+    outline: none;
+    user-select: none;
+    pointer-events: inherit;
+    touch-action: none;
+    -webkit-tap-highlight-color: rgba(0,0,0,0);
+    -webkit-touch-callout: none;
+}
+
+.slider-native-input {
```

Comment:
you don't have to use prefix `slider-`, because all classnames will be encapsulated and you don't have other native input. 
that is correct for all classnames down in this file